.scripts/build.yml

```diff
@@ -1,8 +1,8 @@
 steps:
   - task: NodeTool@0
     inputs:
-      versionSpec: "14.x"
-    displayName: "Use Node 14"
+      versionSpec: "16.x"
+    displayName: "Use Node 16"
```

Comment:
I have changed all the node version as 16.x. because we have used shrinkwrap to resolve the cg issue in testmodeler as we don't directly depend on the problematic package and this depdencies is from external, but shrinkwrap only get supported in npm v7+. Therefore, I upgrade the node version in our pipeline.

Reply:
@joheredi @sarangan12 Please take a look at the change. If no other objections, I will merge this PR tomorrow.